tests/pytests/unit/test_master.py

```diff
@@ -4,7 +4,49 @@
 
 import salt.master
 import salt.utils.platform
-from tests.support.mock import patch
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.fixture
+def master_opts():
+    """
+    Master options
+    """
+    yield salt.config.master_config(None)
+
+
```

Comment:
```suggestion
```
This fixture already exists.
https://github.com/saltstack/salt/blob/0a983597763985ead07bfcbfbc56b74381501ddc/tests/pytests/unit/conftest.py#L24